xdsl/dialects/stablehlo.py

```diff
@@ -43,6 +51,32 @@ def __init__(
         super().__init__(operands=(lhs, rhs), result_types=(result_type,))
 
 
+# endregion
+
+# region Attributes
+
+
+class Precision(StrEnum):
+    """
+    XLA precision for an operand. Has backend specific meaning.
+    """
+
+    DEFAULT = "DEFAULT"
```

Comment:
I believe you could use `auto()` here

Reply:
For some reason it made the string lowercase and my test failed

Reply:
`# |                                    Expected `default`, `HIGH`, or `HIGHEST`.`

Reply:
Yep, documentation seems to say that `auto` lowercases